### PR TITLE
File Server Connector

### PR DESCRIPTION
**Pull request by dilini-muthumala**

### CODE REVIEW

File: file/components/org.wso2.carbon.transport.file/src/main/java/org/wso2/carbon/transport/file/connector/server/util/FileTransportUtils.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+/**
+ * Utility class for File Transport.
+ */
+public class FileTransportUtils {
+
+    private static final Pattern URL_PATTERN = Pattern.compile("[a-z]+://.*");
+    private static final Pattern PASSWORD_PATTERN = Pattern.compile(":(?:[^/]+)@");
+
+    public static FileSystemOptions attachFileSystemOptions(
```
Comment (malakasilva) [inline file/components/org.wso2.carbon.transport.file/src/main/java/org/wso2/carbon/transport/file/connector/server/util/FileTransportUtils.java]: Are we using this anywhere?

Comment (malakasilva) [inline file/components/org.wso2.carbon.transport.file/src/main/java/org/wso2/carbon/transport/file/connector/server/util/FileTransportUtils.java]: Added a minor comment. Please fix.

Comment (dilini-muthumala) [reply file/components/org.wso2.carbon.transport.file/src/main/java/org/wso2/carbon/transport/file/connector/server/util/FileTransportUtils.java]: It's being used at: https://github.com/wso2/carbon-transports/pull/171/files#diff-4afc5380bd67846cd47bee53eeb0a8e6R78

**Review by malakasilva — APPROVED** [review]
All the discussed changes are implemented.

**Pull request merged**